try {
...
// Probeer deze code uit te voeren
// Indien een error opgeworpen wordt, staakt de executie
// en wordt de code in de catch block uitgevoerd
...
} catch (error) {
...
// catch block
...
} finally {
...
// finally block wordt altijd uitgevoerd, ongeacht of er
// een exception opgeworpen wordt of niet
}
